crates/bevy_asset/src/server/mod.rs

```diff
@@ -287,14 +284,17 @@ impl AssetServer {
         self.load_internal(None, path, false, None).await
     }
 
+    /// Performs an async asset load.
+    /// `input_handle` must only be [`Some`] if `should_load` was true when retrieving `input_handle`. This is an optimization to
```

Comment:
```suggestion
    ///
    /// `input_handle` must only be [`Some`] if `should_load` was true when retrieving `input_handle`. This is an optimization to
```

I think otherwise this will be treated as one paragraph, which might not be what you intended